### PR TITLE
Hotfix readme

### PR DESCRIPTION
**Pull request by User12837985123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
...
+
+1. Open in Android Studio
+2. Insert Google Maps API key in P8/DeadCrumbs/app/src/release/res/values/google_maps_api.xml 
+3. Start the application using Android Studio
+
+
+###ASP.NET Core API
+To start the API do the following:
+
+1. Insert a MongoDB configuration file at P8/aspnet-server-1/ on JSon format that contains the following fields "ip", "port", "username", "password" in order to connect to a MongoDB.
```
Comment (KVISDAOWNER) [inline README.md]: The file must be named mongoConnection.json

File: README.md [hunk 2]
```diff
@@ -1 +1,19 @@
...
+# P8
+
+This GitHub repository contains two applications: an Android application written in Kotlin and an ASP.NET Core REST API. The Prototype folder contains two XD files for our GUI prototype, latest.xd being the newest.
+
+##How To Run
+
+###Android Application
+To run the Android application do the following:
+
+1. Open in Android Studio
```
Comment (KVISDAOWNER) [inline README.md]: Specify the folder that must be opened

File: README.md [hunk 3]
```diff
@@ -1 +1,19 @@
...
+2. Insert Google Maps API key in P8/DeadCrumbs/app/src/release/res/values/google_maps_api.xml 
+3. Start the application using Android Studio
+
+
+###ASP.NET Core API
+To start the API do the following:
```
Comment (KVISDAOWNER) [inline README.md]: "To run the API locally do the following:"

**Review by KVISDAOWNER — DISMISSED** [review]
Overall good! I have some small suggestions.

**Review by KVISDAOWNER — APPROVED** [review]

**Review by Sworgon — APPROVED** [review]
Looks alright

**Pull request merged**